src/components/Minimap/index.js

```diff
@@ -45,14 +46,22 @@ const Minimap = ({ data, isLoading, params, rowCount=4 }) => {
     const currentUrl = window.location.pathname
     const areaUrl = areaHits.length && areaHits[0].uri
     const isAreaActive = areaUrl === currentUrl
+    const hitPlural = t({
+      comment: "Pluralization of hit(s) for minimap boxes",
+      message: plural(areaHits.length, {one: "hit", other: "hits"})
+    })
+    const message = t({
+      comment: "Message displayed to jump to specific section within minimap",
+      message: `Jump to ${areaHits.length} ${hitPlural} in this area: ${hitTitles}`
+    })
```

Comment:
I made the messages that I needed translated done outside of the returned data. Is this a better way for me to make the code easier to read?

Reply:
Yes, I like this!